src/kayenta/service/canaryConfig.service.ts

```diff
@@ -88,7 +96,7 @@ export function buildNewConfig(state: ICanaryState): ICanaryConfig {
       groupWeights: {} as { [key: string]: number },
     },
     judge: {
-      name: CanarySettings.defaultJudge,
+      name: CanarySettings.defaultJudge || 'NetflixACAJudge-v1.0',
```

Comment:
(2b) Default `defaultJudge` to 'NetflixACAJudge-v1.0'

Reply:
(which I ended up defaulting in two places rather in the top-level settings file to respect the existing pattern of not setting any defaults there)